dali/pipeline/operator/op_spec.h

```diff
@@ -444,47 +426,49 @@ class DLL_PUBLIC OpSpec {
   }
 
   template <typename T, typename S>
-  inline bool TryGetArgumentImpl(T &result,
-                                 const string &name,
+  bool TryGetArgumentImpl(T &result,
+                                 std::string_view name,
                                  const ArgumentWorkspace *ws,
                                  Index idx) const;
 
   template <typename T, typename S>
-  inline std::vector<T> GetRepeatedArgumentImpl(const string &name) const;
+  std::vector<T> GetRepeatedArgumentImpl(std::string_view name) const;
 
   template <typename S, typename C>
-  inline bool TryGetRepeatedArgumentImpl(C &result, const string &name) const;
+  bool TryGetRepeatedArgumentImpl(C &result, std::string_view name) const;
 
   string schema_name_;
   const OpSchema *schema_ = nullptr;
 
   // the list of arguments, in addition order
   std::vector<std::shared_ptr<Argument>> arguments_;
   // maps names to argument indices
-  std::unordered_map<string, int> argument_idxs_;
+  std::map<string, int, std::less<>> argument_idxs_;
```

Comment:
```suggestion
  std::map<std::string, int, std::less<>> argument_idxs_;
```
You use std:: in other places

Reply:
Find-replace... If there are non-nitpicks from other reviewers I'll update.

Reply:
...actually, there are 10 occurrences `std::map<string, ` vs 31 `std::map<std::string, `, so it's not like it stands out...